shared-bindings/audiofilters/Filter.c

```diff
@@ -38,7 +38,7 @@
 //|            The mix parameter allows you to change how much of the unchanged sample passes through to
 //|            the output to how much of the effect audio you hear as the output.
 //|
-//|         :param Optional[synthio.Biquad] filter: The normalized biquad filter object used to process the signal.
+//|         :param Optional[List[synthio.Biquad]] filters: A list of normalized biquad filter objects used to process the signal.
```

Comment:
You might say something like """The sample is processed sequentially by each filter in turn to give the output samples."""